### PR TITLE
py-tensorflow py-tensorflow-tensorboard : Update to 1.11.0

### PR DESCRIPTION
**Pull request by cjones051073**
#### Description

Updates to 1.11.0 releases

###### Type(s)
- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.14 18A391
Xcode 10.0 10A255 

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-tensorflow-tensorboard/Portfile [hunk 1]
```diff
@@ -17,19 +17,19 @@ long_description    TensorBoard is a suite of web applications for inspecting \
 homepage            https://github.com/tensorflow/tensorboard
 
 if {${python.version} < 30} {
-    master_sites        https://pypi.python.org/packages/d5/fb/80df4eb3234c41beeeb3122b1966effbf08608fb80d2fa74bb72f8da9cb3
-    distname            tensorflow_tensorboard-${version}-py2-none-any
-
-    checksums           rmd160  52c4d0d3bf8856e2ebc87ae9785319e4416ccdce \
-                        sha256  2b8741aed01f35c95b781d25ebf1a84a45c2980746a616a36ade7ed8078d88ee
+    master_sites        https://files.pythonhosted.org/packages/76/f9/e62022d00940e4df9a629d6bfe42eb28907bb35808db62bb9e8b69ea5ef3
```
Comment (yan12125) [inline python/py-tensorflow-tensorboard/Portfile]: You may want to use the hash-free URL: `https://files.pythonhosted.org/packages/cp${python.version}/t/tensorflow`

File: python/py-tensorflow-tensorboard/Portfile [hunk 2]
```diff
@@ -17,19 +17,19 @@ long_description    TensorBoard is a suite of web applications for inspecting \
...
+    distname            tensorboard-${version}-py2-none-any
+    
+    checksums           rmd160  64f9323d502b46b0ec85969030af2bfb729698f6 \
+                        sha256  c5adadd4ba12aa8a036d46591c17ceda93033df12ab1ea8e265194645f597ed6 \
+                        size    3027414
 } else {
-    master_sites        https://pypi.python.org/packages/68/82/6b00646b7275c3da90107333df7fbecaae72a9bfd3139fedd11686084d90
-    distname            tensorflow_tensorboard-${version}-py3-none-any
+    master_sites        https://files.pythonhosted.org/packages/9b/2f/4d788919b1feef04624d63ed6ea45a49d1d1c834199ec50716edb5d310f4
```
Comment (yan12125) [inline python/py-tensorflow-tensorboard/Portfile]: ditto

File: python/py-tensorflow-tensorboard/Portfile (hunk 1 above)
Comment (cjones051073) [reply python/py-tensorflow-tensorboard/Portfile]: Thanks for the hint, that is a nice simplification.

**Comment by cjones051073** [comment]
I have update this PR a little further with two additional commits.

bazel : Update to 0.17.2 release.

py-tensorflow : Add a new native variant that performs a build from source.

The benefit of the build from source option is the build by default is tuned for the local CPU, so is able to better target the capabilities of that CPU, by using `march=native` in the build options. On my local CPU here this allows the use of AVX2 and FMA instructions, and gives a modest speed improvement.

**Comment by cjones051073** [comment]
Yes, bazel is an interesting build system.... It starts an background deamon and unless you start bazel in 'batch' that deamon remains running after the command that spawn it is long finished.  That 4 hour pause was just the time it took for that daemon to timeout and die, I think. The tcl system command was just wating for it...

Note that the '--batch' option is depreciated, so whilst it works now it will go in the future. Instead we are supposed to call `bazel shutdown`.  I will have a go to see if this work as well...

**Comment by cjones051073** [comment]
turns out fixing the `py37` build was not esay, so I have removed this option again pending upstream support for this version.

**Pull request merged**